src/seedsigner/gui/screens/screen.py

```diff
@@ -1322,3 +1323,90 @@ class MainMenuScreen(LargeButtonScreen):
     title_font_size: int = 26
     show_back_button: bool = False
     show_power_button: bool = True
+
+
+
+@dataclass
+class BaseKeyDetailsScreen(WarningEdgesMixin, ButtonListScreen):
+    """
+    Standardized base class for displaying key details.
+    
+    Args:
+        title: Screen title
+        key_label: Label for the key (e.g., "Xpub", "Private Key")
+        key_value: The actual key value to display
+        derivation_path: BIP32 derivation path
+        fingerprint: BIP32 master fingerprint
+        has_passphrase: Whether seed has passphrase (for warning styling)
+        button_data: List of ButtonOption objects
+        icon_name: FontAwesome icon for the key line (defaults to 'x')
```

Comment:
There's no harm here, but we don't document every Screen like this. It's more important to just document the unique args.